--- a/icons/Irixium/index.theme
+++ b/icons/Irixium/index.theme
@@ -2,7 +2,7 @@
 Name=Irixium
 Comment=Remixed NsCDE
 Example=start-here
-Inherits=hicolor,breeze
+Inherits=breeze,hicolor
 
 # KDE Specific Stuff
 DisplayDepth=32
@@ -10,17 +10,17 @@
 LockOverlay=lock_overlay
 ZipOverlay=zip_overlay
 DesktopDefault=24
-DesktopSizes=16,22,32,48
+DesktopSizes=16,22,24,32,48
 ToolbarDefault=24
-ToolbarSizes=16,22,32,48
+ToolbarSizes=16,22,24,32,48
 MainToolbarDefault=24
-MainToolbarSizes=16,22,32,48
+MainToolbarSizes=16,22,24,32,48
 SmallDefault=16
 SmallSizes=16
 PanelDefault=48
-PanelSizes=16,22,32,48
+PanelSizes=16,22,24,32,48
 DialogDefault=22
-DialogSizes=16,22,32,48
+DialogSizes=16,22,24,32,48
 
 # Directory list
 Directories=16x16/actions,16x16/animations,16x16/apps,16x16/categories,16x16/devices,16x16/emblems,16x16/emotes,16x16/mimetypes,16x16/places,16x16/status,22x22/actions,22x22/animations,22x22/apps,22x22/categories,22x22/devices,22x22/emblems,22x22/emotes,22x22/mimetypes,22x22/places,22x22/status,24x24/actions,24x24/apps,24x24/categories,24x24/devices,24x24/emblems,24x24/emotes,24x24/mimetypes,24x24/places,24x24/status,32x32/actions,32x32/animations,32x32/apps,32x32/categories,32x32/devices,32x32/emblems,32x32/emotes,32x32/mimetypes,32x32/places,32x32/status,48x48/actions,48x48/animations,48x48/apps,48x48/categories,48x48/devices,48x48/emblems,48x48/emotes,48x48/mimetypes,48x48/places,48x48/status,256x256/places
@@ -273,7 +273,5 @@
 [256x256/places]
 Context=Places
 Size=256
-MinSize=56
-MaxSize=512
-Type=Scalable
-
+Type=Fixed
+
